README.md

```diff
@@ -0,0 +1,18 @@
+# OBS Portable
+
+OBS Portable is a portable version of [OBS Studio for Windows](https://obsproject.com/) packaged in [PortableApps.com Format](http://portableapps.com/about/what_is_a_portable_app).
+
+[Visit OBS Portable Homepage at PortableApps.com](https://portableapps.com/node/53266)
+
+__[Download OBS Portable 19.0.2 Development Test 1][download]__
```

Comment:
I'd do this a bit more generic so we don't need to change it with every new version. Like: [Download Lastest OBS Portable]
I don't know if we really need a readme because I already link to the PortableApps.com homepage.